src/Storages/Statistics/CountMinSketchStatistics.h

```diff
@@ -0,0 +1,43 @@
+#pragma once
+
+#include <Storages/Statistics/Statistics.h>
+
+#include "config.h"
+
+#if USE_DATASKETCHES
+
+#include <count_min.hpp>
+
+namespace DB
+{
+
+class CountMinSketchStatistics : public IStatistics
+{
+public:
+    CountMinSketchStatistics(const SingleStatisticsDescription & stat_, DataTypePtr data_type_);
+
+    Float64 estimateEqual(const Field & value) const;
+
+    void serialize(WriteBuffer & buf) override;
+    void deserialize(ReadBuffer & buf) override;
+
+    void update(const ColumnPtr & column) override;
+
+    bool checkType(const Field & f);
+
+private:
+    static constexpr auto HASH_COUNT = 8uz;
```

Comment:
Future readers will wonder how you arrived at these magic constants?

Kindly add a comment that describes your reasoning and the impact of the parameters on the error rate and size.